easybuild/easyconfigs/q/QuickPIC/quickpic-makefile.patch

```diff
@@ -0,0 +1,49 @@
+Patch for makefile and make.GF_OPENMPI so HDF5 and json-fortran is being found
+instead of hard-coded paths
+Author: J. Sassmannshausen (Imperial College London/UK)
+diff --git a/QuickPIC-OpenSource-776fcffaaf9e170fbae80bfe54abc95b1f84dfca.orig/source/make.GF_OPENMPI b/QuickPIC-OpenSource-776fcffaaf9e170fbae80bfe54abc95b1f84dfca/source/make.GF_OPENMPI
+index b2f08f0..c1538cf 100644
+--- a/QuickPIC-OpenSource-776fcffaaf9e170fbae80bfe54abc95b1f84dfca.orig/source/make.GF_OPENMPI
++++ b/QuickPIC-OpenSource-776fcffaaf9e170fbae80bfe54abc95b1f84dfca/source/make.GF_OPENMPI
+@@ -1,6 +1,6 @@
+ # name of the compiler 
+-JSON_LIB = /usr/local/jsonfortran-gnu-6.3.0/lib/
+-JSON_INC = /usr/local/jsonfortran-gnu-6.3.0/lib/
++JSON_LIB = ${EBROOTJSONMINFORTRAN}/lib
++JSON_INC = ${EBROOTJSONMINFORTRAN}/include
+ 
+ FC_GF_OPENMPI = mpif90 -c -I$(JSON_INC) -fopenmp
+ CC_GF_OPENMPI = mpicc -c -O3 -std=c99
+@@ -14,11 +14,11 @@ FORMAT_FIXED_GF_OPENMPI = -ffixed-form
+ 
+ # hdf5 libraries 
+ 
+-HDF5_LIB =/usr/local/hdf5/lib
+-HDF5_INC =/usr/local/hdf5/include
++HDF5_LIB =${EBROOTHDF5}/include
++HDF5_INC =${EBROOTHDF5}/lib
```

Comment:
@sassy-crick I think you mixed these up ;-)
```suggestion
+HDF5_LIB =${EBROOTHDF5}/lib
+HDF5_INC =${EBROOTHDF5}/include
```
Note that due to this change, the checksum will also have to be recalculated for the patch file.

Reply:
Yes, that was me correcting one thing and got it wrong in the other place. Thanks for finding that! All done now.